src/pip/_internal/operations/check.py

```diff
@@ -67,6 +63,7 @@ def check_package_set(package_set, should_ignore=None):
     """
     if should_ignore is None:
         def should_ignore(name):
+            # type: (str) -> bool
             return False
```

Comment:
So we can remove all this, right?

Reply:
Do you mean to ask if we can remove the type annotation? If I try to remove it, mypy complains with `error: Function is missing a type annotation`

Reply:
I mean to remove the whole `if should_ignore is None` block

Reply:
Ohh, you meant if we can remove the if condition and the function inside it, I think we can.

Reply:
Removed in https://github.com/pypa/pip/pull/8322/commits/8fc3feea369c443200f2437016a747fde9567f5d